project.json

```diff
@@ -0,0 +1,5 @@
+{
+  "build":"./gradlew assembleRelease; ./gradlew uploadArchives",
+  "test":"./gradlew :spatialconnect:connectedCheck; ./gradlew :SpaconApp:connectedCheck",
```

Comment:
add  `&&` between the unit test and integration test commands it basically mean only execute the second one if first command succeeded